### PR TITLE
Contributing document

### PR DESCRIPTION
**Pull request by ethervoid**
Document explaining how to contribute to the project.

### CODE REVIEW

**Comment by ethervoid** [comment]
Please @rafatower check this out :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# General
+
+Every new feature (as well as bugfixes) should come through a [github pull requests](https://help.github.com/articles/using-pull-requests) and should have test cases for it.
+
+Go [here](https://github.com/CartoDB/odbc_fdw/tree/master/test#connectors-integration-tests) to learn how the test framework works and how to add new tests to the project.
+
+# Bugs and features
+
+Before you start working on a new feature, we encourage you to create an issue in github explaining what are you going to do in order to help you and especially to avoid duplicate work.
```
Comment (rafatower) [inline CONTRIBUTING.md]: > Before you start working on a new feature

"Before you start working on a new **bug or feature**..."

> we encourage you to create an issue

I'd say "we encourage you to first look up the open issues list to check there's no duplicate. If your issue is not described there, please create one..."

> what are you going

"what you are going to do" instead

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+There are several rules you should follow when a new pull request is created:
+
+- Title has to be descriptive. If you are fixing a bug don't use the ticket title or number.
```
Comment (rafatower) [inline CONTRIBUTING.md]: I'd say "don't use **just** the issue number"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+3. [Submitting contributions](#submitting-contributions)
+
+---
+
+# General
+
+Every new feature (as well as bugfixes) should come through a [github pull requests](https://help.github.com/articles/using-pull-requests) and should have test cases for it.
+
+Go [here](https://github.com/CartoDB/odbc_fdw/tree/master/test#connectors-integration-tests) to learn how the test framework works and how to add new tests to the project.
+
```
Comment (rafatower) [inline CONTRIBUTING.md]: Slightly out of scope: I'd add a small section about submitting good bug reports. Also a template comes in handy.

**Comment by ethervoid** [comment]
Plase @rafatower , I've made some changes to the document. Could you please review them? Thank you

**Pull request merged**